lib/rubocop/granite/inject.rb

```diff
@@ -11,7 +9,7 @@ def self.defaults!
         path = CONFIG_DEFAULT.to_s
         hash = ConfigLoader.load_file(path)
         config = Config.new(hash, path).tap(&:make_excludes_absolute)
-        puts "configuration from #{path}" if ConfigLoader.debug?
+        Rails.logger.debug { "configuration from #{path}" } if ConfigLoader.debug?
```

Comment:
What do you think about using global logger?
```suggestion
        Granite::Form.config.logger.debug { "configuration from #{path}" } if ConfigLoader.debug?
```

I do not see much usage of loggers in the repository but anyway it would be good to keep it generic.

Reply:
This was an automatic change by rubocop. We can discuss further changes as separate tickets @mbie